reference_manual.py

```diff
@@ -0,0 +1,2 @@
+def color_pair_to_string(major_color, minor_color):
```

Comment:
The section "New Feature Request" in the README.md. It's about printing all the color and number combinations. Add that feature.

Reply:
I have committed the change please check.